commerce_coordinator/settings/local.py

```diff
@@ -83,11 +83,11 @@
 )
 BACKEND_SERVICE_EDX_OAUTH2_KEY = os.environ.get(
     'BACKEND_SERVICE_EDX_OAUTH2_KEY',
-    'commerce_coordinator-backend-service-key'
```

Comment:
This will break ALL current installs on local. As well as the provisioning scripts... Its why this action hasnt been taken.

Reply:
Check out 22f93e4fdf244a0b78c105dc7ab73d77eac6c587, where I correct the provisioning scripts! I think it would be good to make local consistent with stage/prod.